src/java.desktop/unix/classes/sun/awt/screencast/ScreencastHelper.java

```diff
@@ -28,6 +28,7 @@
 import sun.awt.UNIXToolkit;
 import sun.security.action.GetPropertyAction;
 
+import javax.swing.Timer;
```

Comment:
Is there a reason you can't use java.util.Timer ? 
Ideally [*] AWT classes should not depend on Swing classes since already Swing classes depend on AWT classes .. 

[*] yes, I know we already have such dependencies but only for very good reasons.

Reply:
Of course we can use java.util.Timer. Swing timer is just more convenient to use. Updated.